styles/components/_alert.scss

```diff
@@ -0,0 +1,25 @@
+.alert {
+  display: flex;
+  border-width: 2px;
+  border-style: solid;
+  border-color: none;
+  border-radius: 0.25rem;
```

Comment:
The client wanted a border or a background color? I thought we were talking about something like toast notifications (https://www.google.com/search?q=toast+notification&rlz=1C1CHBD_plPL929PL929&sxsrf=ALeKk01-6cxYgZ_nQuzfMmaq7i7HxHbTzQ:1613057945748&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjUuu6OleLuAhVPQkEAHY1cCaUQ_AUoAXoECBIQAw&biw=1777&bih=841). If we aren't sure, maybe you could present to the client.